elementary/monitor/alerts/source_freshness.py

```diff
@@ -0,0 +1,101 @@
+import datetime
+from typing import Optional
+
+from elementary.clients.slack.schema import SlackMessageSchema
+from elementary.monitor.alerts.alert import Alert
+from elementary.utils.log import get_logger
+from elementary.utils.time import try_parse_time
+
+logger = get_logger(__name__)
+
+
+class SourceFreshnessAlert(Alert):
+    TABLE_NAME = "alerts_source_freshness"
+
+    def __init__(
+        self,
+        unique_id: str,
+        snapshotted_at: Optional[str],
+        max_loaded_at: Optional[str],
+        max_loaded_at_time_ago_in_s: Optional[float],
+        source_name: str,
+        identifier: str,
+        freshness_error_after: str,
+        freshness_warn_after: str,
+        freshness_filter: str,
+        path: str,
+        **kwargs,
+    ) -> None:
+        super().__init__(**kwargs)
+        self.unique_id = unique_id
+        self.snapshotted_at = try_parse_time(snapshotted_at) if snapshotted_at else None
+        self.max_loaded_at = try_parse_time(max_loaded_at) if max_loaded_at else None
+        self.max_loaded_at_time_ago_in_s = max_loaded_at_time_ago_in_s
+        self.source_name = source_name
+        self.identifier = identifier
```

Comment:
Do you know when identifier exists for a source and when name exists? I wonder what's the difference between them

Reply:
<img width="421" alt="image" src="https://user-images.githubusercontent.com/30181361/195208353-e964eda1-a6a7-4d5d-b831-c3c2a62c2d78.png">

From what I'm seeing throughout most of the code,
it seems like `name` is either just a wrapper for `identifier`, or that it was deprecated for it.
Anyhow, from what I've sent, you can see that `identifier` is always present as it fallbacks to `name`.